tsconfig.json

```diff
@@ -0,0 +1,73 @@
+{
+    "compilerOptions": {
+        /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+        /* Basic Options */
+        // "incremental": true,                   /* Enable incremental compilation */
```

Comment:
Codacy found an issue: [Comment not allowed](https://app.codacy.com/gh/P2P-Develop/TopDomainChecker/pullRequest?prid=6364165)